From e4260562edd937591a9ca246138e5f29cfd804da Mon Sep 17 00:00:00 2001
From: Raviteja Laggyshetty <quic_rlaggysh@quicinc.com>
Date: Tue, 30 Apr 2024 11:44:27 +0530
Subject: [PATCH 1/4] PENDING: dt-bindings: interconnect: Add EPSS L3 DT
 binding on SA8775P

Add Epoch Subsystem (EPSS) L3 interconnect provider binding
on sa8775p SoCs.

Change-Id: I149e6020ee86c43cdf8bfd3819e526b83f57df77
Signed-off-by: Raviteja Laggyshetty <quic_rlaggysh@quicinc.com>
Upstream-Status: Pending
---
 Documentation/devicetree/bindings/interconnect/qcom,osm-l3.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/interconnect/qcom,osm-l3.yaml b/Documentation/devicetree/bindings/interconnect/qcom,osm-l3.yaml
index 21dae0b92819..de2c59ddc94a 100644
--- a/Documentation/devicetree/bindings/interconnect/qcom,osm-l3.yaml
+++ b/Documentation/devicetree/bindings/interconnect/qcom,osm-l3.yaml
@@ -33,6 +33,8 @@ properties:
               - qcom,sm6375-cpucp-l3
               - qcom,sm8250-epss-l3
               - qcom,sm8350-epss-l3
+              - qcom,sa8775p-epss-l3-cl0
+              - qcom,sa8775p-epss-l3-cl1
           - const: qcom,epss-l3
 
   reg:
-- 
2.17.1

